### PR TITLE
Updates for Puppet 4 compatibility

### PR DESCRIPTION
**Pull request by petems**
* Use scope lookup instead of variables

### CODE REVIEW

**Comment by petems** [comment]
@ericsakowski Mind having a look at this? 👍

File: .travis.yml [hunk 1]
```diff
@@ -12,9 +12,10 @@ env:
...
     - PUPPET_GEM_VERSION="~> 3.7"
+    - PUPPET_GEM_VERSION="~> 4.0"
 
 matrix:
   exclude:
 
 notifications:
-  email: corey@logicminds.biz
\ No newline at end of file
+  email: corey@logicminds.biz
```
Comment (logicminds) [inline .travis.yml]: oh snap! we need change this to the new owner.

File: templates/iloconfig.erb [hunk 2]
```diff
@@ -1,52 +1,52 @@
...
   <RIB_INFO MODE="write">
-<% if dhcp == false -%>
+<% if scope['dhcp'] == false -%>
     <MOD_NETWORK_SETTINGS>
       <SPEED_AUTOSELECT VALUE = "Y"/>
-<% if !shared -%>
+<% if scope['shared'] == false -%>
       <SHARED_NETWORK_PORT VALUE="N"/>
 <% end -%>
-<% if shared -%>
```
Comment (logicminds) [inline templates/iloconfig.erb]: @petems why are we using scope?  I though scope was discouraged.

**Comment by logicminds** [comment]
@petems do this conflict with #6?  Can we merge both?

**Comment by petems** [comment]
@logicminds Nah, #6 has commits from this PR, but I wanted to make sure they were ok before doing the next one. So merge this one first, then the other after a rebase from my side 👍

File: templates/iloconfig.erb (hunk 2 above)
Comment (petems) [inline templates/iloconfig.erb]: `lookup.scope` was deprecated, vanilla scope is ok. 

https://docs.puppet.com/puppet/latest/reference/lang_template_erb.html#accessing-puppet-variables

**Comment by logicminds** [comment]
@petems thanks for updating the templates to the new standard.  These are very old.

**Pull request merged**